blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/FilteringTreePruner.java

```diff
@@ -25,7 +28,7 @@ public TreePruner accept(Object node, Property prop) {
 
         // for merge properties, the current restrictions on the property names should
         // still apply to the child TreePruner
-        if (prop.merge)
+        if (prop.merge && child != null)
```

Comment:
Use of `merge` in the 'Cause' `@Exported(name="cause", merge = true)` caused this NPE. This check fixes it but perhaps this should go upstream?